packages/common/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ All logs must start with the format: [x.y.z] - yyyy-mm-dd
 
 ## [Unreleased]
 
+## [1.4.0] - 2022-08-17
+### Changed
+- Support new templates mono-repo structure. (#1236)
```

Comment:
What does it mean @stwiname? Would it make immediate sense for devs?

Reply:
This is an internal change that doesn't impact devs.